frontend/src/metabase-lib/v2/limit.ts

```diff
@@ -0,0 +1,15 @@
+import * as MetabaseLibLimit from "cljs/metabase.lib.limit";
+import type { Query, Limit } from "./types";
+
+export function currentLimit(query: Query): Limit {
+  return MetabaseLibLimit.current_limit(query);
+}
+
+export function limit(query: Query, limit: Limit): Query {
+  return MetabaseLibLimit.limit(query, limit);
+}
+
+export function hasLimit(query: Query) {
+  const limit = currentLimit(query);
+  return typeof limit === "number" && limit > 0;
+}
```

Comment:
Not sure how cool we are about adding thin helpers in a wrapping library. Should this better be done in CLJS? 🤔

Reply:
yeah, I think it makes sense to add to cljs.

Reply:
I'm not philosophically opposed to a bit of TS logic, but in this case the function is useful in Clojure too, so it may as well live there.